src/Combobox/index.js

```diff
@@ -119,12 +141,10 @@ const Combobox = ({
     );
   }
 
-  const hasSelectedItem = !!selectedItem;
-
   return (
     <div
       className={cc(["nds-combobox", { "nds-combobox--active": isOpen }])}
-      {...getComboboxProps()}
+      {...getComboboxProps(triggerProps)}
```

Comment:
These prop getters from `downshift` return an object of props to spread onto the given component. They all take an argument of extra props to spread, making it easy to compose `downshift` with props returned from `useLayer`